.github/workflows/test.yml

```diff
@@ -18,14 +18,10 @@ jobs:
 
       - uses: ./.github/workflows/yarn
 
-      - name: Run tests
-        run: 'yarn test:ci'
-
-      - name: Comment in failing tests
-        uses: mattallty/jest-github-action@v1.0.3
-        if: failure()
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Annotations and coverage report
+        uses: ArtiomTr/jest-coverage-report-action@v2
```

Comment:
Is there a reason we are not using Coveralls? It seems that we already have some projects on there: https://coveralls.io/github/safe-global

Reply:
I don't see the point of depending on a 3rd-party service for something available locally.